posthog/models/plugin.py

```diff
@@ -197,6 +198,12 @@ def __str__(self) -> str:
 
 
 class PluginConfig(models.Model):
+    class Meta:
+        indexes = [
+            models.Index(fields=["web_token"]),
+            models.Index(fields=["enabled"]),
+        ]
```

Comment:
Disclaimer: This is the general theory, the practice needs to be tested with `EXPLAIN ANALYZE`

Reply:
Luckily postgres indexes are one of the few things we can add and remove as needed :). I went for simple indices since I didn't know what shape the queries will take, and didn't want to overanalyse before this was a problem. Many simple indices also take up less space than multiple combined ones, so I considered this a fair tradeoff 🤷 

Already the queries for /decide and /web_js/id/token/ are quite different in what indices they use. Currently: `id` & `web_token` & `enabled` ... vs ... `team_id` & `enabled`.